internal/app/machined/pkg/system/services/etcd.go

```diff
@@ -147,6 +147,8 @@ func (e *Etcd) Runner(r runtime.Runtime) (runner.Runner, error) {
 		env = append(env, "ETCD_UNSUPPORTED_ARCH=arm64")
 	}
 
+	env = append(env, "ETCD_CIPHER_SUITES=TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305") //nolint:lll
```

Comment:
what is the effect of this line on etcd security/performance?

Reply:
ref (one of them): https://github.com/etcd-io/etcd/issues/10304

Reply:
* AES 128/256 with RSA/ECDSA for CPU with AES-IN
* chacha-poly with RSA/ECDSA for ARM

https://blog.cloudflare.com/do-the-chacha-better-mobile-performance-with-cryptography/

etcd cluster works inside the tales cluster, and it does not support to connect external etcd nodes. We can set less choice but with better performance.